Added missing JSDOC param name

zraineri opened the pull request:
Added param originalVersionString to ServerVersion constructor

zraineri committed:
Added missing JSDOC param name

Added param originalVersionString to ServerVersion constructor

bigmontz commented:
Hi @zraineri 

thanks for you contribution. For your PR be able to be reviewed, you need to read and sign Neo4j's CLA (https://neo4j.com/developer/cla/).

zraineri commented:
@bigmontz I apologize for the late reply. The CLA has been read and signed

zraineri committed:
Merge branch 'neo4j:4.4' into patch-1

zraineri committed:
Edited JSDOC to show optional param

originalVersionString in the constructor method of ServerVersion is treated as optional in the codebase (line 71). I updated the JSDOC param to match this.

zraineri committed:
Merge branch 'neo4j:4.4' into patch-1

zraineri committed:
Merge branch 'neo4j:4.4' into patch-1

zraineri committed:
Merge branch 'neo4j:4.4' into patch-1
